config/openshift copy/kustomization.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
Is the `openshift copy` folder still needed?

Reply:
how was that not deleted is a mystery to me 😅